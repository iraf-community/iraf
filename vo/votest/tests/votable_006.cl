#
#  Print VOTable column information

# Set the test description string.
votest.descr = "TABLES support: Print VOTable column info"

# Convert the data$logical to a local path.
s1 = data_url // "/usno-b.xml"
s2 = "file://" // data_path // "/usno-b.xml"
s3 = "file:///localhost" // data_path // "/usno-b.xml"


fcache init
reset clobber = yes

# Create the output table name.
string out
out = "/tmp/out_006"
if (access (out) == yes) {
    delete (out, verify-, >& "dev$null")
}
;

# Execute the test commands.
print ("\nLogical Path:  data$usno-b.xml")		# logical path
tcopy ("data$usno-b.xml", out, verb-)
tinfo ("data$usno-b.xml,"//out)
delete (out)

print ("\nHTTP URI:  " // s1)				# remote http URI
tcopy (s1, out, verb-)
tinfo (s1//","//out)
delete (out)

print ("\nFile URI:  " // s2)				# file URI
tcopy (s2, out, verb-)
tinfo (s2//","//out)
delete (out)

print ("\nFile URI:  " // s3)				# file URI
tcopy (s3, out, verb-)
tinfo (s3//","//out)
delete (out)

