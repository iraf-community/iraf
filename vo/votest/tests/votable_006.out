
Logical Path:  data$usno-b.xml
# data$usno-b.xml
  65 rows written to table
  16 columns defined
 188 header parameters written to table
 215 records allocated for header parameters
  16 space allocated for column descriptors
table type:  fits
# /tmp/out_006
  65 rows written to table
  16 columns defined
 133 header parameters written to table
 133 records allocated for header parameters
  17 space allocated for column descriptors
table type:  text  explicit column definitions

HTTP URI:  http://iraf.noao.edu/votest//usno-b.xml
# http://iraf.noao.edu/votest//usno-b.xml
  65 rows written to table
  16 columns defined
 188 header parameters written to table
 215 records allocated for header parameters
  16 space allocated for column descriptors
table type:  fits
# /tmp/out_006
  65 rows written to table
  16 columns defined
 133 header parameters written to table
 133 records allocated for header parameters
  17 space allocated for column descriptors
table type:  text  explicit column definitions

File URI:  file:///iraf/iraf/vo/votest/data//usno-b.xml
# file:///iraf/iraf/vo/votest/data//usno-b.xml
  65 rows written to table
  16 columns defined
 188 header parameters written to table
 215 records allocated for header parameters
  16 space allocated for column descriptors
table type:  fits
# /tmp/out_006
  65 rows written to table
  16 columns defined
 133 header parameters written to table
 133 records allocated for header parameters
  17 space allocated for column descriptors
table type:  text  explicit column definitions

File URI:  file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml
# file:///localhost/iraf/iraf/vo/votest/data//usno-b.xml
  65 rows written to table
  16 columns defined
 188 header parameters written to table
 215 records allocated for header parameters
  16 space allocated for column descriptors
table type:  fits
# /tmp/out_006
  65 rows written to table
  16 columns defined
 133 header parameters written to table
 133 records allocated for header parameters
  17 space allocated for column descriptors
table type:  text  explicit column definitions
